core/models/notification.py

```diff
@@ -0,0 +1,84 @@
+# Standard Library
+import datetime
+
+# Third-Party Imports
+from django.db import models
+from django.utils.timesince import timesince
+
+# App Imports
+from core import constants
+
+
+class Notifications(models.Model):
+    """
+    A model representing the a notification object
+    Nomenclature based on https://www.w3.org/TR/notifications/
+    Generalized Format for notifications body::
+        <actor> <verb> <time>
+        <actor> <verb> <target> <time>
+        <actor> <verb> <action_object> <target> <time>
+
+    Examples::
+        <user> <reached level 60> <1 minute ago>
+        <user2> <commented on> <url/post/1d> <2 hours ago>
+        <user> <started follow> <user2> <8 minutes ago>
+        <user> <closed> <issue 70> on <topic> <about 2 hours ago>
+    """
+
+    title = models.CharField(
```

Comment:
I also see that the model fields specified in the story are not the same as you have here.. could you reach out to the FPC and have the story description updated.

Reply:
I will reach out and have the story updated 👍

Reply:
Let's tailor the generalized format to our specific use case as much as possible. Nice job